config/default.json

```diff
@@ -19,8 +19,9 @@
     },
     "enketo": {},
     "env": {
-      "domain": "http://localhost:8989",
-      "sysadminAccount": "no-reply@getodk.org"
+      "domain": "http://localhost",
+      "sysadminAccount": "no-reply@getodk.org",
+      "port": "8989"
```

Comment:
i guess my question is, why this split? can we leave the `8989` where it already was?